packages/firestore-types/index.d.ts

```diff
@@ -37,7 +37,13 @@ export interface PersistenceSettings {
   experimentalTabSynchronization?: boolean;
 }
 
-export type LogLevel = 'debug' | 'error' | 'silent';
+export type LogLevel =
+  | 'debug'
+  | 'error'
+  | 'silent'
+  | 'warn'
+  | 'info'
+  | 'verbose';
```

Comment:
Are you planning to make it public? If so, you have to update `packages/firebase/index.d.ts`

Reply:
@wu-hui @schmidt-sebastian

Reply:
Pinged on chat